conda/cli/main_clean.py

```diff
@@ -65,6 +72,66 @@ def configure_parser(sub_parsers):
     p.set_defaults(func=execute)
 
 
+# work-around for python bug on Windows prior to python 3.2
```

Comment:
@kalefranz re 2: given that this function is called from a loop for every single file, your suggestion will lead to significant performance degradation. It might be more sensible to make ntfsutils a prerequisite instead and simply use a one line import if this code block bothers you.

I am happy for you to take ownership of this issue and close this PR if you want to refactor the code.